### PR TITLE
Fix EarlGrey CocoaPods project and add a Travis target for it. 

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: Demo/EarlGreyExample/Podfile [hunk 1]
```diff
@@ -41,4 +49,4 @@ post_install do |installer|
     end
     configure_for_earlgrey(installer, PROJECT_NAME, test_target, test_target + '.xcscheme', var_args)
   end
-end
+end
```
Comment (khandpur) [inline Demo/EarlGreyExample/Podfile]: no new line?

**Comment by tirodkar** [comment]
I updated this PR. We also fix some comments in the gem and add a way to run the CocoaPods project with the local gem from travis.

**Comment by tirodkar** [comment]
The build is fixed here. I'm not providing a file path for the `rake install`but instead entering it and installing the gem.

**Comment by sid-github** [comment]
LGTM

**Pull request merged**